keywords.robot

```diff
@@ -1886,7 +1886,7 @@ Remove Entry From List
 
 Get Secure Boot Configuration Submenu Construction
```

Comment:
I overlooked this kwd. This should be gone from the keywords.robot.

Reply:
I will remove it.

Reply:
Ok I see you already did that https://github.com/Dasharo/open-source-firmware-validation/commit/0567475d5abec3b7f8a769f2d122054ca59a45db